scripts/test/lint.py

```diff
@@ -86,6 +86,20 @@ def run_cmd(cmd):
     return out
 
 
+class InvalidDescriptionLength(Lint):
+    max_length = 175
+    name = "the description length is invalid"
+    recommendation = "the description must be limited to 174 characters maximum."
```

Comment:
This is wrong, the max length is 175 and not 174. I think we should use `max_length` here to ensure they are on sync.
```suggestion
    recommendation = f"the description must be limited to {max_length} characters maximum."
```